controller/array_action/array_mediator_svc.py

```diff
@@ -301,22 +303,20 @@ def _get_fcmaps(self, volume_name, endpoint_type):
         filter_value = '{0}_vdisk_name={1}'.format(endpoint_type, volume_name)
         return self.client.svcinfo.lsfcmap(filtervalue=filter_value).as_list
 
-    def validate_supported_capabilities(self, capabilities):
-        logger.debug("validate_supported_capabilities for "
-                     "capabilities : {0}".format(capabilities))
-        # Currently, we only support one capability "SpaceEfficiency"
-        # The value should be: "thin/thick/compressed/deduplicated"
-        if (capabilities and capabilities.get(
-                config.CAPABILITIES_SPACEEFFICIENCY).lower() not in
-                [config.CAPABILITY_THIN, config.CAPABILITY_THICK,
-                 config.CAPABILITY_COMPRESSED,
-                 config.CAPABILITY_DEDUPLICATED]):
-            logger.error("capability value is not "
-                         "supported {0}".format(capabilities))
-            raise controller_errors.StorageClassCapabilityNotSupported(
-                capabilities)
-
-        logger.info("Finished validate_supported_capabilities")
+    def validate_supported_space_efficiency(self, space_efficiency):
+        logger.debug("validate_supported_space_efficiency for "
+                     "space efficiency : {0}".format(space_efficiency))
+
+        if (space_efficiency and space_efficiency.lower() not in
+                [config.SPACE_EFFICIENCY_THIN, config.SPACE_EFFICIENCY_THICK,
+                 config.SPACE_EFFICIENCY_COMPRESSED,
+                 config.SPACE_EFFICIENCY_DEDUPLICATED]):
+            logger.error("space efficiency type value is not "
```

Comment:
[again](https://github.com/IBM/ibm-block-csi-driver/pull/212#discussion_r528718931), no need for `type`

Reply:
done